app/views/users/show.html.erb

```diff
@@ -1,14 +1,158 @@
+                    </div>
+                </div>
+                    <div class="modal-footer">
+                      <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                      <%= f.submit 'Submit', class: 'btn btn-primary' %>
+                    </div>
+                      <%	end  %>
+              </div>
+            </div>
+          </div>
+        </div>
+    </p>
+  </div>
+</div>
+<div id="wrap">
+  <div id="header">
+    <h1><%= @user.first_name  %> <%= @user.last_name %></h1>
+    <h2><%= @user.headline %></h2>
+  </div>
+  <div id="bio">
+    <div id="about">
+      <div id="photo-bg">
+        <%= image_tag @user.image_url, size: '170x170' if !@user.image_url.nil? %>
+      </div>
+      <h3>About Me</h3>
+      <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec id elit non mi porta gravida at eget metus. Aenean lacinia bibendum nulla sed consectetur. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
+      <h4>Status: <span> On line</span> </h4>
+    </div>
+    <div id="contact-details">
+      <h3>Contact Me</h3>
+      <p>Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh. Please get in touch:</p>
+      <h5>Email: <span><%= @user.email %></span></h5>
+      <h5>Twitter: <span><%= @user.twitter %></span></h5>
+      <h5>Slack: <span><%= @user.slack %></span></h5>
+      <h5>Github: <span><%= @user.github %></span></h5>
+      <h5>Industry: <span><%= @user.summary %></span></h5>
+
+    </div>
+  </div>
+</div>
+<div class="line"></div>
 
 <p><%= link_to 'Authorize app to use your Linkedin information',
                '/auth/linkedin', class: "btn btn-primary"  %></p>
-</div>
+
+<section class="cards">
+
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title">Interest</h3>
+            <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+            <a href="#" class="btn btn-primary">Go somewhere</a>
+          </div>
+        </div>
+      </div>
+      <div class="col-sm-6">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title">What I'm working on</h3>
+            <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+            <a href="#" class="btn btn-primary">Go somewhere</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+        <div class="container">
+          <div class="resume">
+            <%#= file_field_tag @user.resume %>
+            <%= image_tag @user.resume.to_s, size: '170x170' if !@user.resume.nil? %>
+
```

Comment:
@chantal66 Everything else looks great. Is the code starting on line 147 through 158 still in use? If not, please take it out before merge. Thanks!